site/docs/getting-started/azureAuthorize.jsx

```diff
@@ -1,19 +1,51 @@
 import React from "react";
 
-export const AzureAuthorizeComponent = () => {
-    const ourAppId = "42cb0c6c-dab0-411f-9c21-16d5a2b1b025";
-    const redirectUri = "https://eyrcnmuzzyriypdajwdk.supabase.co/functions/v1/azure-dpc-oauth";
-    const resourceId = "https://storage.azure.com";
+const SETTINGS = {
+    appId: "42cb0c6c-dab0-411f-9c21-16d5a2b1b025",
+    redirectUri: "https://eyrcnmuzzyriypdajwdk.supabase.co/functions/v1/azure-dpc-oauth",
+    resourceId: 'https://storage.azure.com',
+    responseType: 'code',
+    stateKey: 'state',
+    storageKey: 'docs_est:azure_oauth_state'
+}
+
+const generateAuthorizeUrl = (theirTenant) => {
+    const state = crypto.randomUUID()
+    sessionStorage.setItem(SETTINGS.storageKey, state);
```

Comment:
This is the MAJOR update here. We need to store this off and validate it down below.